test/jobs/fetch_faculty_data_job_test.rb

```diff
@@ -71,7 +71,6 @@ def expected_building_names
         "Crean Hall",
         "DeMille Hall",
         "Digital Media Arts Center",
-        "Dodge College",
```

Comment:
When you say "live data", you mean the test is connecting to a live feed or other resource? If so, I agree: bad. Use mock/stubs.

I looked at the test and it looked like it was using VCR which I assume function like a deterministic fixture. But I've never used VCR before.

If needs to be fixed, I'm ok putting it off and adding an action item. Actually, I already assigned myself an action item to install webmock which will answer my first question.